tests/pinkopy/test_helper.py

```diff
@@ -0,0 +1,52 @@
+import requests_mock
+import uuid
+
+
+def get_uuid():
+    return str(uuid.uuid4())
+
+
+def mock_session(base_session, service='http://' + get_uuid(),
+                 user=get_uuid(), pw=get_uuid(), token=get_uuid(),
+                 content_type='application/json', clients=get_uuid()
+                 ):
+    get_payload = {
+        'App_GetClientPropertiesResponse': {
+            'clientProperties': clients
+        }
+    }
+    post_payload = {
+        'DM2ContentIndexing_CheckCredentialResp': {
+            '@token': token
+        }
+    }
+    headers = {
+        'Accept': content_type,
+        'Content-type': content_type
+    }
+    with requests_mock.mock() as m:
+        m.get(service + '/Client', json=get_payload)
```

Comment:
I thought it was weird that you were calling them payloads, but now I see the issue. This may pass tests, but it, I believe, misunderstands what is happening. We aren't sending a payload with a GET requests. In fact, I'm not even sure that requests allows for this. It would be ignored by the server. I believe the `get_payload` should be `get_response` and should be the response yielded from `m.get(urljoin(service, 'Clients'))`.